boards/arm/frdm_k64f/doc/index.rst

```diff
@@ -241,16 +241,29 @@ configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`.
 Early versions of this board have an outdated version of the OpenSDA bootloader
 and require an update. Please see the `DAPLink Bootloader Update`_ page for
 instructions to update from the CMSIS-DAP bootloader to the DAPLink bootloader.
+The version 0254 is known to work well to flash and debug.
 
 .. tabs::
 
    .. group-tab:: OpenSDA DAPLink Onboard (Recommended)
 
-        Install the :ref:`pyocd-debug-host-tools` and make sure they are in your search
-        path.
+	Install the :ref:`linkserver-debug-host-tools` and make sure they are in your
+	search path.  LinkServer works with the default CMSIS-DAP firmware included in
+	the on-board debugger.
+
+        Linkserver is the default for this board, ``west flash`` and ``west debug`` will
```

Comment:
When LinkServer is the default runner, this doc needs some additional changes.  On the tab "OpenSDA JLink Onboard", change this sentence:
> Add the arguments ... when you invoke west build to override the default runner from pyOCD to J-Link.

to
> Add the arguments ... when you invoke west build to override the default runner to J-Link.